examples/xlnet/xlnet_generation_main.py

```diff
@@ -14,28 +14,22 @@
 """Example of building XLNet language model for sample generation.
 """
 
+import os
 import argparse
 import torch
 import sentencepiece as spm
 
-import xlnet
-import xlnet.model.decoder
-from texar.modules import TopKSampleEmbeddingHelper, TopPSampleEmbeddingHelper
+import texar as tx
+
+from utils import data_utils
 
 parser = argparse.ArgumentParser()
 parser.add_argument('--checkpoint', type=str, default=None,
-                    help="Checkpoint to load model weights from. Use "
-                         "`--pretrain_checkpoint` instead if loading XLNet "
-                         "pretrained checkpoint.")
-parser.add_argument('--pretrain_checkpoint', type=str,
-                    default="pretrained/xlnet_cased_L-24_H-1024_A-16/"
-                            "xlnet_model.ckpt",
-                    help="XLNet pretrained model checkpoint. Ignored if "
-                         "'--checkpoint' is specified.")
-parser.add_argument('--pretrain_model_dir', type=str,
-                    default="pretrained/xlnet_cased_L-24_H-1024_A-16",
-                    help="The directory of pretrained model, for loading "
-                         "vocabulary, etc.")
+                    help="Checkpoint to load model weights from.")
+parser.add_argument("--pretrained_model_name", type=str,
```

Comment:
Same here. Avinash wrote this file so it didn't follow the same naming convention. Let's unify them.